src/python/visclaw/plotclaw.py

```diff
@@ -60,7 +60,15 @@ def plotclaw(outdir='.', plotdir='_plots', setplot = 'setplot.py',
     if plotdata.num_procs is None:
         plotdata.num_procs = int(os.environ.get("OMP_NUM_THREADS", 1))
 
-    if plotdata.parallel and plotdata.num_procs != 1:
+    # Make sure plotdata.parallel is False in some cases:
+
+    if plotdata.num_procs == 1:
+        plotdata.parallel = False   # not doing in parallel in this case
+
+    if type(setplot) is not str:
+        plotdata.parallel = False   # cannot call plotclaw from shell
```

Comment:
Please don't silently override the user specification.  I would much prefer an assertion here.  Otherwise one will wonder why the parallel plotting doesn't actually speed things up, and we'll waste a lot of time digging down to this line to diagnose it.